Makefile

```diff
@@ -1,3 +1,8 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(dir $(mkfile_path))
```

Comment:
Can you use `CURDIR` here? https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html